api/repositories/org_repository_test.go

```diff
@@ -553,6 +556,7 @@ var _ = Describe("OrgRepository", func() {
 							"key-two": "value-two",
 						},
 					))
+					Expect(updatedCFOrg.Spec.DisplayName).To(Equal(*orgNewName))
```

Comment:
We need to also check that the org record name is updated (I am referring to the `It` on line 526, the UI is not letting me comment on that line)